lib/renaissance/bids/bids.ex

```diff
@@ -0,0 +1,53 @@
+defmodule Renaissance.Bids do
+  import Ecto.Query
+  alias Renaissance.{Repo, Bid}
+  alias Renaissance.Helpers.Adapt
+
+  def place_bid(params) do
+    params = Adapt.format_amount(params, "amount")
```

Comment:
so this is basically creating it...do we need to have it use a special name instead of the standard ones?

Reply:
Resolved in 15d9389.